lib/ex_sdp/utils.ex

```diff
@@ -40,4 +40,30 @@ defmodule ExSDP.Utils do
       _string_not_0_nor_1 -> {:error, :string_not_0_nor_1}
     end
   end
+
+  @spec parse_sprop_parameter_sets(binary) ::
+          {:error, :invalid_sprop_parameter_sets} | {:ok, %{sps: binary, pps: binary}}
+  def parse_sprop_parameter_sets(string) do
+    result =
+      String.split(string, ",", parts: 2)
+      |> then(fn list -> List.zip([[:sps, :pps], list]) end)
+      |> Enum.map(fn {type, encoded} ->
+        with {:ok, decoded} <- Base.decode64(encoded) do
+          {type, <<0, 0, 0, 1>> <> decoded}
+        else
+          :error -> {type, nil}
+        end
+      end)
+      |> Enum.into(%{})
```

Comment:
1. Instead of `Enum.map` + `Enum.into` you can use `Map.new`
2. In the suggestion I've applied the reversed order from a previous suggestion.
3. It's a matter of personal preference, but I wouldn't use `with` here
4. I've left the append of `<<0,0,0,1>>` prefix, but I've made a comment about it separately
```suggestion
      |> Map.new(fn {encoded, type} ->
        decoded =
          case Base.decode64(encoded) do
            {:ok, decoded} -> <<0, 0, 0, 1>> <> decoded
            :error -> nil
          end
        
        {type, decoded}
      end)
```